### ISSUE TITLE
Support for fog-core 2.3.0

### ISSUE THREAD

**Issue by micwoj92** [fog/fog-google#586]
When running fog this module gives error
```
/usr/lib/ruby/3.0.0/rubygems/specification.rb:2288:in `raise_if_conflicts': Unable to activate fog-google-1.19.0, because fog-core-2.3.0 conflicts with fog-core (< 2.3) (Gem::ConflictError)
	from /usr/lib/ruby/3.0.0/rubygems/specification.rb:1419:in `activate'
	from /usr/lib/ruby/3.0.0/rubygems/specification.rb:1457:in `block in activate_dependencies'
	from /usr/lib/ruby/3.0.0/rubygems/specification.rb:1439:in `each'
	from /usr/lib/ruby/3.0.0/rubygems/specification.rb:1439:in `activate_dependencies'
	from /usr/lib/ruby/3.0.0/rubygems/specification.rb:1421:in `activate'
	from /usr/lib/ruby/3.0.0/rubygems.rb:286:in `block in activate_bin_path'
	from /usr/lib/ruby/3.0.0/rubygems.rb:285:in `synchronize'
	from /usr/lib/ruby/3.0.0/rubygems.rb:285:in `activate_bin_path'
	from /usr/bin/fog:25:in `<main>'

```

**Comment by micwoj92** [comment]
Still same

**Comment by dajabe** [comment]
I'm having an issue similar where I can't update to fog-core >= 2.3

https://github.com/fog/fog-google/pull/622

There is a PR here to upgrade the requirements of fog-core to < 2.5

Looks like the tests just haven't been given the computer time to run. What do we need to get this done?

**Comment by Temikus** [comment]
@dajabe released in 1.24.0, see #628
